packages/adapter-components/src/fetch/element/id_utils.ts

```diff
@@ -121,18 +146,21 @@ const computeElemIDPartsFunc =
         if (part.custom !== undefined) {
           return part.custom(part)(entry)
         }
-        return getNameMapping({
-          name: _.get(entry, part.fieldName) ?? '',
-          nameMapping: part.mapping,
-          customNameMappingFunctions,
-        })
+        const fieldValue = getFieldValue(entry, part.fieldName)
+        return fieldValue !== undefined
+          ? getNameMapping({
+              name: fieldValue,
+              nameMapping: part.mapping,
+              customNameMappingFunctions,
```

Comment:
I don't mind changing it to

```suggestion
        getNameMapping({
              name: fieldValue ?? '',
              nameMapping: part.mapping,
              customNameMappingFunctions,
```